### PR TITLE
[SPARK-14050] [ML] Add multiple languages support and additional methods for Stop Words Remover

### PR DESCRIPTION
**Pull request by mengxr**
## What changes were proposed in this pull request?

This PR continues the work from #11871 with the following changes:
- load English stopwords as default
- covert stopwords to list in Python
- update some tests and doc
## How was this patch tested?

Unit tests.

Closes #11871

cc: @burakkose @srowen

### CODE REVIEW

File: mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala [hunk 1]
```diff
@@ -110,37 +60,37 @@ class StopWordsRemover(override val uid: String)
...
 
   /**
-   * whether to do a case sensitive comparison over the stop words
+   * Whether to do a case sensitive comparison over the stop words.
    * Default: false
    * @group param
    */
   val caseSensitive: BooleanParam = new BooleanParam(this, "caseSensitive",
-    "whether to do case-sensitive comparison during filtering")
+    "whether to do a case-sensitive comparison over the stop stop words")
```
Comment (jkbradley) [inline mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala]: "stop stop" --> "stop"

File: python/pyspark/ml/feature.py [hunk 2]
```diff
@@ -1763,28 +1763,22 @@ class StopWordsRemover(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadabl
...
 
     @keyword_only
     @since("1.6.0")
-    def setParams(self, inputCol=None, outputCol=None, stopWords=None,
-                  caseSensitive=False):
+    def setParams(self, inputCol=None, outputCol=None, stopWords=None, caseSensitive=False):
         """
-        setParams(self, inputCol="input", outputCol="output", stopWords=None,\
-                  caseSensitive=false)
+        setParams(self, inputCol="input", outputCol="output", stopWords=None, caseSensitive=false)
```
Comment (jkbradley) [inline python/pyspark/ml/feature.py]: doc: inputCol, outputCol should not have defaults

**Comment by jkbradley** [comment]
Should there be a unit tests which iterates through StopWordsRemover.supportedLanguages and tests loading all & checking they are non-empty?

Other than those small items, this looks good to me

**Comment by jkbradley** [comment]
LGTM pending tests

**Comment by mengxr** [comment]
Merged into master and branch-2.0.

**Pull request closed without merging**